OrbitGl/GraphTrack.cpp

```diff
@@ -32,15 +33,26 @@ void GraphTrack::Draw(GlCanvas* canvas, PickingMode picking_mode) {
   }
 
   float track_z = GlCanvas::kZValueTrack;
-  float text_z = GlCanvas::kZValueText;
 
   Box box(pos_, Vec2(size_[0], -size_[1]), track_z);
 
   // Draw label with graph value at current mouse position.
-  const Color kLabelTextColor(255, 255, 255, 255);
+  const Color kWhiteColor(255, 255, 255, 255);
+  const Color kBlackColor(0, 0, 0, 255);
   double graph_value = GetValueAtTime(time_graph_->GetCurrentMouseTimeNs());
-  canvas->GetTextRenderer().AddText(std::to_string(graph_value).c_str(), canvas->GetMouseX(), y0,
-                                    text_z, kLabelTextColor);
+
+  Vec2 arrow_box_size(50., layout.GetTextBoxHeight());
```

Comment:
What is the arrow box? I don't understand why this is called "arrow". 

And why is it okay to use a constant for the width here? If you need the constant, make it a named constant where the name reflects the usage.

Reply:
Sorry for that. My plan is to make something like an arrow box, but will come in a future PR. I renamed it.

It's clearly not ok :)